mycelo/loadbot/bot.go

```diff
@@ -87,15 +100,24 @@ func Start(ctx context.Context, cfg *Config) error {
 			clientIdx++
 			client := cfg.Clients[clientIdx%len(cfg.Clients)]
 			group.Go(func() error {
-				return runTransaction(ctx, lg, sender, nonce, cfg.Verbose, cfg.SkipGasEstimation, client, recipient, cfg.Amount)
+				cfg := txConfig{
```

Comment:
I'm not a fan of shadowing when the types and objects involved are unrelated.  I would suggest `txCfg` as a name instead (though it looks ugly for some reason).